.travis.yml

```diff
@@ -56,4 +59,6 @@ before_install:
 
 install: composer update
 
-before_script: mkdir Snc && ln -s ../ Snc/RedisBundle
+before_script:
+  - mkdir Snc && ln -s ../ Snc/RedisBundle
+  - php clientbuilder/build_redis_client.php
```

Comment:
I am not sure why it is necessary to have that in the CI.

IMHO, the class should be generated at cache warmup then dump into var/cache/snc-redis-bundle and then loaded thanks to a custom autoloader. It's basically what doctrine-bundle is doing for entities.

What do you think?

Reply:
I think that could work only when using DI, but be aware that `PhpredisClientFactory` doesn't depend on the container.
Also, in my first approach I was thinking to check the generated file in each build in order to detect any change based on the installed Redis version, but I think it doesn't have sense now that the installed versions are set explicitly at Travis config.